.github/workflows/test.yml

```diff
@@ -21,12 +21,17 @@ jobs:
         name: cachix-action
         file: test.nix
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
-    - name: Test public cache no pushing
+    - name: Test public cache no signing key
       uses: ./
       with:
         name: cachix-action
         file: test.nix
         signingKey: ''
+    - name: Test public cache no pushing
+      uses: ./
+      with:
+        name: cachix-action
+        skipNixBuild: true
```

Comment:
I don't think this fails if it tries to build default.nix or does it?

Reply:
It does, since `default.nix` doesn't exist. It's not the best assertion, but it's something :)

Reply:
So we have the opposite problem.
That makes this pr a slight improvement, even if it may have to change.